### PR TITLE
Add Github Actions build

### PR DESCRIPTION
**Pull request by gliptak**
https://github.com/gliptak/docker-lambda/pull/1/checks?check_run_id=1166239653

### CODE REVIEW

**Comment by gliptak** [comment]
@mhart please review

**Comment by mhart** [comment]
Can you elaborate?

**Comment by gliptak** [comment]
After this PR is merged, the build can be updated to build all ```Dockerfile```s in the project ensuring that build process is successful (running on all PRs and ```master``` commits)

**Comment by mhart** [comment]
There's already a build script to build all the images, you can use that if you like: https://github.com/lambci/docker-lambda/blob/master/base/build-all.sh

**Comment by gliptak** [comment]
@mhart running https://github.com/lambci/docker-lambda/blob/master/base/build-all.sh runs out-of-space https://github.com/gliptak/docker-lambda/runs/1171597286?check_suite_focus=true

**Comment by mhart** [comment]
Yeah, there are 40 images now – I guess GH Actions can't do this out of the box

**Comment by gliptak** [comment]
Separating into two builds works
https://github.com/gliptak/docker-lambda/pull/1/checks?check_run_id=1172828830
https://github.com/gliptak/docker-lambda/pull/1/checks?check_run_id=1172828834

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build Docker images
+      run: |
+        cd base && ./build-all-build.sh
```
Comment (mhart) [inline .github/workflows/ci-build.yml:17]: Probably a little cleaner to do:
```suggestion
    - name: Build Docker images
      working-directory: base
      run: ./build-all-build.sh
```

Comment (gliptak) [reply .github/workflows/ci-build.yml]: updated

**Comment by gliptak** [comment]
Runs are green
https://github.com/gliptak/docker-lambda/pull/1/checks?check_run_id=1173396166
https://github.com/gliptak/docker-lambda/pull/1/checks?check_run_id=1173396168

**Comment by gliptak** [comment]
@mhart any other updates you would need to see?

**Comment by mhart** [comment]
All good, thanks @gliptak !

**Pull request merged**